CHANGELOG.rst

```diff
@@ -21,6 +21,12 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 * Cookietemple's GitHub Actions now print lint and create results colored
 
+* Added requirements.txt and requirements_dev.txt (Python projects) to blacklisted sync files
```

Comment:
maybe it is easier if we always add the handle in the front of such changes

-> cli-python, web-python: added requirements [...]

Reply:
Yeah, good idea